### PR TITLE
fix: align otel attributes

### PR DESCRIPTION
**Pull request by StarpTech**
## Motivation and Context

This PR aligns several OTEL attributes. Previously, we added client attributes in a server context.

#### Checklist

- [ ] run `make test`
- [ ] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: pkg/trace/wrap_handler.go [hunk 1]
```diff
@@ -14,13 +15,18 @@ func WrapHandler(wrappedHandler http.Handler, componentName attribute.KeyValue,
 	// Don't trace health check requests or favicon browser requests
 	opts = []otelhttp.Option{
 		otelhttp.WithFilter(RequestFilter),
+		// TODO: Use router path as span name, high cardinality should be avoided
```
Comment (Aenimus) [inline pkg/trace/wrap_handler.go:18]: Will this be address in another PR?

**Review by Aenimus — APPROVED** [review]

File: pkg/trace/wrap_handler.go (hunk 1 above)
Comment (StarpTech) [reply pkg/trace/wrap_handler.go:18]: Nope, it's ok for now. It just doesn't follow best practices. Ideally, we would have span names like `/operations/{name*}` or `/webhooks/{name}`.

**Review by jensneuse — APPROVED** [review]
